internal/cloud/realm/secrets_test.go

```diff
@@ -46,5 +47,20 @@ func TestSecrets(t *testing.T) {
 				assert.Equal(t, []realm.Secret{secret}, secrets)
 			})
 		})
+
+		t.Run("should delete the app secret", func(t *testing.T) {
```

Comment:
[optional/style] personally I'd just nest this further under `should create a secret` and avoid the `var secret realm.Secret` declaration at the top